cfme/automate/explorer/domain.py

```diff
@@ -289,6 +289,8 @@ def create(self, name=None, description=None, enabled=None, cancel=False):
             k: v
             for k, v in {'name': name, 'description': description, 'enabled': enabled}.items()
             if v is not None}
+        #   Workaround for solving automation issue.
```

Comment:
@mfalesni, what's your opinion on this fix ?

Reply:
@izapolsk actually there is a change in UI, now we don't need to load the saved advance search option to delete it so there was a test case failure.

Reply:
@izapolsk : In the above case the name field doesn't get filled so I have to perform the fill action again.

Reply:
@anurag03, I understand but I'm not sure that this is the best way to fix this issue.

btw, please add this form  to google doc where we collect pages with such old forms.

Reply:
@psav This issue has to do with old forms. We need consensus on workarounds. I'm for making workarounds such as this with a `# FIXME FORM` comment nearby for easy identification later. Thoughts?

Reply:
Consider putting this in `before_fill()` or after_fill()` for the impacted views instead of a separate fill call. Then any tests directly filling will automatically include it.